andela/pulse-mobile-app
# [#156371588] Add redux setup

### What does this PR do?
Implements  redux set up.

### Description of Task to be completed?
 - Added redux boilerplate
 - Added redux persist of store to async storage

### How should this be manually tested?
- Run the application using `react-native run-android` or `react-native run-ios`  to see the changes.
- Enable debugging Js remotely on your terminal 
- Navigate within the app and watch the state changes logged on console.

### Any background context you want to provide?
This is basic project set up, hoping to be basis to start building more features.

#### What are the relevant pivotal tracker stories?
[#156371588](https://www.pivotaltracker.com/story/show/156371588)
### Screenshot
![redux-persist](https://user-images.githubusercontent.com/15029943/38544459-2d7c2460-3cb0-11e8-9acc-e289afcccdd6.gif)
